### ISSUE TITLE
switched to this project, dev works, but build doesn't

### ISSUE THREAD

**Issue by husayt** [prisma/nuxt-prisma#56]
Before I was using prisma with nuxt directly, Dev was working fine, but to build a server bundle I had my own setup and some after build script
```
copy .env  .output\server

rem move myrpisma outside from node_modules
move .output\server\node_modules\myprisma .output\server\myprisma

Rem this file is not needed for linux
del .output\server\myprisma\client\query_engine-windows.dll.node

rem copy database file to dist folder
copy ..\prisma\dev.db .output\server\myprisma\client

powershell -Command "(gc .output\server\package.json) -replace '0.0.0', 'file:../myprisma' -replace 'win32-x64-msvc', 'linux-x64-gnu' | Out-File -encoding ASCII .output\server\package.json"

rem Replace inside `.output/server` all 'myprisma/client' to '../myprisma/client/index.js'
node fileReplacer.js
```

Obviously that was painful, so I switched to using `nuxt-prisma` module  to have smoother use. I managed to get dev working with
```
nuxi dev --dotenv .env.local
```
but can't even get local build running properly. 
I get build completed with these warnings 
![image](https://github.com/user-attachments/assets/d84acfd6-9788-4211-8701-1114eb53040d)
then when i run with  `node .output/server/index.mjs` it can't find database, but if I copy sqlite database file to output folder and run like this it works:
![image](https://github.com/user-attachments/assets/c78dc7f8-c56b-4565-9ffb-f0a1b29214fc)

Is there anything I am missing, i expect it to copy database file to output folder. Or is this as expected?

**Comment by husayt** [comment]
>This is not expected. Could you check if [this](https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/prisma-nuxt-module#resolving-typeerror-failed-to-resolve-module-specifier-prismaclientindex-browser) helps resolve the issue?

I have tried that and now get this error when running from build and it crashes completely in browser
![image](https://github.com/user-attachments/assets/49fd3f8f-e2db-4eee-a8b3-2d0935208d57)

**Comment by ankur-arch** [comment]
Thanks for getting back. 

> I have tried that and now get this error when running from build and it crashes completely in browser

I haven't seen the exact message before. I'd appreciate it if you provide me with a small reproduction of the issue.

**Comment by ankur-arch** [comment]
Thanks @husayt, could you kindly provide a reproduction I can use to replicate the exact errors? Additionally, is there any custom configuration I should be aware of?

**Comment by husayt** [comment]
Yes, you are right. I am trying to come up with a small reproduction and will let you know once it is ready.

Thank you

**Comment by AmanVarshney01** [comment]
Sadly, we are deprecating this module. You can read more about the explanation [here](https://github.com/prisma/nuxt-prisma/pull/112)

How to use Nuxt with Prisma 7 - https://www.prisma.io/docs/guides/nuxt
